src/styles/navbar.css

```diff
@@ -0,0 +1,40 @@
+.nav {
+  width: 87.5rem;
+  height: 5.938rem;
+  margin: 0 0 2.375rem;
+  padding: 1.438rem 6.188rem 1.688rem 6.25rem;
+  background-color: #fff;
+  border-bottom: solid 1px #e8e8e8;
+}
+
+.app-name {
+  width: 15rem;
+  margin: 0 3rem 0.8rem 0.12rem;
+  font-family: Montserrat, serif;
+  font-size: 1.875rem;
+  font-weight: bold;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.App-link {
+  margin: 0.5rem 1rem;
+  font-family: Montserrat, serif;
+  font-size: 0.813rem;
+  letter-spacing: 2px;
+  color: #121212;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+.opacity-5 {
+  opacity: 0.5;
+}
+
+.Oval {
+  width: 2.813rem;
+  height: 2.813rem;
+  margin: 0 0 0 41.5rem;
+  padding: 0.875rem;
+  border: solid 1px #e8e8e8;
+}
```

Comment:
- [ ] It would be better if we fix the horizantal scroll at the bottom of the page,  this has made the website unresponsive. Here is a screenshot of how it shows on my system
![Screenshot from 2022-08-19 14-53-58](https://user-images.githubusercontent.com/46388113/185648830-d31927c6-70a8-44e3-ab4d-31f7ace446b5.png)